lldb/include/lldb/Symbol/UnwindTable.h

```diff
@@ -64,7 +64,7 @@ class UnwindTable {
 private:
   void Dump(Stream &s);
 
-  void Initialize();
+  void Initialize(bool force = false);
```

Comment:
This needs a comment explaining what you are forcing.

Reply:
Yeah I'm not thrilled with the terminology I used.  But I had `Initialize` which assumed no unwind sources had been discovered, and `Update` called after a SymbolFileVendor is added to the Module which looks for any un-set unwind sources and checks to see if they can now be found.  That duplication didn't thrill me.